src/iuse.cpp

```diff
@@ -6075,6 +6192,10 @@ static std::string photo_quality_name( const int index )
 
 int iuse::einktabletpc( player *p, item *it, bool t, const tripoint &pos )
 {
+    if( p->is_mounted() ){
+        p->add_msg_if_player( m_info, _( "You cannot use a mop while mounted." ) );
```

Comment:
Mop? What mop is this message talking about?

Btw. it would probably be better to use generic messages (maybe include the item name), like the message for being underwater does. It's easier for the translators and "more" correct. An iuse function may be triggered by any item, so telling the player that "using a shovel is not possible" while they clearly activated a foo (item from mod that happens to have the shovel-iuse) may confuse them.